pkg/sloop/store/untyped/partition.go

```diff
@@ -60,9 +60,17 @@ func GetTimeRangeForPartition(partitionId string) (time.Time, time.Time, error)
 	return oldestTime, newestTime, nil
 }
 
-func TestHookSetPartitionDuration(partDuration time.Duration) bool {
+func GetAgeOfPartitionInHours(partitionId string) (float64, error) {
+	timeForPartition, err := GetTimeForPartition(partitionId)
+	if err != nil {
+		return 0, err
+	}
+
+	return float64((time.Now().Sub(timeForPartition)) / 3600000000000), nil
```

Comment:
consider applying a name over this constant - it might be good to calculate this value (i.e. minutes * seconds * milliseconds)